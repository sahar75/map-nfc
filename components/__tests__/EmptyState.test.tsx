import React from "react";
import TestRenderer from "react-test-renderer";
import EmptyState from "../EmptyState";

describe("EmptyState", () => {
  it("should render correctly with a title", () => {
    const tree = TestRenderer.create(
      <EmptyState title="No Data Available" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should render correctly with an empty title", () => {
    const tree = TestRenderer.create(<EmptyState title="" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
