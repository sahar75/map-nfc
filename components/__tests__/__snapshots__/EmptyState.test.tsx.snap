// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState should render correctly with a title 1`] = `
<View
  className="justify-center items-center px-4"
>
  <Text
    className="text-xl text-center font-psemibold mt-2"
  >
    No Data Available
  </Text>
</View>
`;

exports[`EmptyState should render correctly with an empty title 1`] = `
<View
  className="justify-center items-center px-4"
>
  <Text
    className="text-xl text-center font-psemibold mt-2"
  />
</View>
`;
